src/components/PoolCreator/Wizard/Wizard.js

```diff
@@ -166,14 +166,20 @@ export const Wizard = (props) => {
 
     // create pool contract on the blockchain
     // Get blockchain data after tx confirms, then update pool object
+
+    const args = [
```

Comment:
why did you remove the pool owner address? don't we need it to allow changes to the pool?

Reply:
the second argument was for the node operator actually. we put there the owner just temporary. We know who's the owner cause it's he's the one who sends the tx